replica/backup.go

```diff
@@ -77,7 +73,7 @@ func (rb *Backup) OpenSnapshot(snapID, volumeID string) error {
 
 func (rb *Backup) assertOpen(id, volumeID string) error {
```

Comment:
Why do we need `assertOpen()`? This function restricts that we can only operate one snapshot for each `Backup` object. 
e.g. If I use the backup object (it's `deltaOps` in backupstore) to open snapshot A (`OpenSnapshot()`), then later I could read snapshot A only (`ReadSnapshot()`), or check the existence of snapshot A only (`HasSnapshot()`). It means we cannot use this backup object to check/read snapshot B.